proxy/round_tripper/proxy_round_tripper.go

```diff
@@ -194,7 +194,11 @@ func (rt *roundTripper) RoundTrip(originalRequest *http.Request) (*http.Response
 
 			res, err = rt.timedRoundTrip(roundTripper, request, logger)
 			if err != nil {
-				logger.Error("route-service-connection-failed", zap.Error(err))
+				logger.Error(
+					"route-service-connection-failed",
+					zap.String("endpoint", request.URL.String()),
```

Comment:
Good idea for improving logging. 

Can you make it more clear that this is the route service URL by replacing "endpoint" with something like "route-service-endpoint"?

Reply:
Good catch, adjusted both

Reply:
wow that was fast 🔥